mathics/core/expression.py

```diff
@@ -682,26 +679,25 @@ def evaluate(
                 if hasattr(expr, "options") and expr.options:
                     evaluation.options = expr.options
 
-                # This calls evaluate_next. This routine implements a single
-                # step in the evaluation, and determines if a fixed point
```

Comment:
I tend to stay away from fancy words like "fixed point" when simple words like "will not change" will work. 

If the focus were on program correctness or proving mathematical properties of a program then a technical term would be useful. 

Most programmers don't think in terms of fixed points or idempotence, and to understand this code you really don't need to know about that.